src/libs/ReportUtils.ts

```diff
@@ -805,7 +805,7 @@ function isGroupPolicy(report: OnyxEntry<Report>): boolean {
 /**
  * Whether the provided report belongs to a Control or Collect policy
  */
-function isPaidGroupPolicy(report: OnyxEntry<Report>): boolean {
+function isPaidGroupPolicy(report: OnyxEntry<Report> | EmptyObject): boolean {
```

Comment:
Revert. Use `null`

Reply:
Oh, I see that we use EmptyObject in many places. Updated